### PR TITLE
Chore/handle main rejection

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso @ahmedalsudani please review

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -50,12 +50,16 @@ app.on("window-all-closed", ()=>{
 });
 
 process.on("uncaughtException", (err)=>{
-  platform.writeTextFileSync(platform.getUncaughtErrorFileName(), require("util").inspect(err));
+  var obj = Object.assign({ type: "Exception" }, err);
```
Comment (ahmedalsudani) [inline main.js]: @fjvallarino is it possible for `err` to have a member `type`?

**Comment by fjvallarino** [comment]
@ahmedalsudani updated

**Comment by ahmedalsudani** [comment]
:+1:

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**